src/formatter.rs

```diff
@@ -28,13 +30,27 @@ pub fn layer<S>() -> impl Layer<S>
 where
     S: Subscriber + for<'a> LookupSpan<'a>,
 {
-    tracing_subscriber::fmt::layer()
-        .event_format(EventsFormatter)
-        .fmt_fields(FieldsFormatter)
+    crate::builder().layer()
 }
 
 /// A formatter that formats tracing-subscriber events into logfmt formatted log rows.
-pub struct EventsFormatter;
+pub struct EventsFormatter {
+    pub(crate) with_level: bool,
+    pub(crate) with_target: bool,
+    pub(crate) with_span_name: bool,
+    pub(crate) with_span_path: bool,
+}
```

Comment:
This is technically a breaking change because users could do

```rust
let formatter = EventsFormatter;
```

which would break now since it gains private fields.

So have to remember to ship a new major version (`0.3.0` it seems)